server/routes/koala.router.js

```diff
@@ -1,18 +1,30 @@
+const { Router } = require('express');
```

Comment:
I think this needs to be `const express = require('express');`

Reply:
const { Router } = require('express');
const express = require('express');
const koalaRouter = express.Router();

Here's the full list of imports. I think this was added in automatically with VS code, so maybe we keep it so it doesn't keep auto putting it in.